### DISCUSSION TITLE
顶栏布局修改没有添加“番剧”的按钮吗？

### DISCUSSION THREAD

**Discussion by 1736952126** [the1812/Bilibili-Evolved#1938] [提问 / Q&A]
不知道是我没找到还是确实没有。
如图：从弹性空白一到搜索框都没有“番剧”的选项

![image](https://user-images.githubusercontent.com/61899163/121642541-403b6580-cac3-11eb-9a1f-a8b3feae8db0.png)

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/121645485-d4f39280-cac6-11eb-95f0-669422d75b5f.png)

**Reply by 1736952126** [reply]
订阅处只能查看自己的追番而不能转到bilibili原定的番剧 主页
![image](https://user-images.githubusercontent.com/61899163/121673643-afc14d00-cae3-11eb-8dee-a78b68728979.png)
![image](https://user-images.githubusercontent.com/61899163/121673682-ba7be200-cae3-11eb-87aa-f9570ee4a051.png)

**Reply by the1812** [reply]
番剧主页那不就在主站里

**Reply by Geeyun-JY3** [reply]
可不可以理解为想要一个原版顶栏有的入口
![image](https://user-images.githubusercontent.com/49396421/122646606-91e29080-d152-11eb-9d91-cb346b1c7269.png)

**Reply by the1812** [reply]
原来如此

**Reply by RingingVolcano** [reply]
> 可不可以理解为想要一个原版顶栏有的入口
> ![image](https://user-images.githubusercontent.com/49396421/122646606-91e29080-d152-11eb-9d91-cb346b1c7269.png)

感觉没啥必要 放在主站里也好找
![LBHJ$@_WZ()_9BKJ~1K4( E](https://user-images.githubusercontent.com/72140353/123606905-f4611e00-d82f-11eb-97d7-d62989b9afe1.png)

**Reply by Geeyun-JY3** [reply]
> 感觉没啥必要 放在主站里也好找

但要移鼠标，耗时更长，步骤多啊，哪有直接放在顶栏，需要时点击进入快

**Reply by RingingVolcano** [reply]
> > 感觉没啥必要 放在主站里也好找
> 
> 但要移鼠标，耗时更长，步骤多啊，哪有直接放在顶栏直接点击进入快

按官方手机app的分区布局 番剧是和其他区放一起的 都放主站下面也算符合逻辑
而且据我了解作者平常不怎么看番（x
![K%_VAH SZNIJ~(Q WX}78QP](https://user-images.githubusercontent.com/72140353/123611180-ead9b500-d833-11eb-99db-7e28023a8a85.png)

**Reply by Geeyun-JY3** [reply]
> 按官方手机app的分区布局 番剧是和其他区放一起的 都放主站下面也算符合逻辑

手机端是手机端，网页端是网页端，手机端怎么布局说明不了什么。
更何况网页端的原版顶栏就有这个入口，遵循官方的逻辑不更应该加这么个入口了？
![image](https://user-images.githubusercontent.com/49396421/122646606-91e29080-d152-11eb-9d91-cb346b1c7269.png)

> 而且据我了解作者平常不怎么看番（x

和作者看不看番有啥关系。更何况作者也实现了挺多自己原本没有的需求啊。

**Reply by RingingVolcano** [reply]
> 和作者看不看番有啥关系。更何况作者也实现了挺多自己原本没有的需求啊。

单独放外面什么必要 又不是难找。。。

**Reply by Geeyun-JY3** [reply]
> 单独放外面什么必要 又不是难找。。。

从来就不是难找的问题。
顺着你这逻辑，不如问问 B 站在顶栏放这么个入口（单独放外面）有什么必要？本脚本自定义顶栏**可以**将稍后再看**单独放外面**又有什么必要？也不是很难找啊。

这不过是一个需求而已。你觉得「入口不难找」已经能满足你的需求。那楼主的需求呢？「入口不难找」就是 TA 的需求吗？
说实话，我自己也没这需求，我也只是顺着楼主逻辑推演而已。稍微推演共情后，我觉得需求应该是更快地打开 [番剧主页](https://www.bilibili.com/anime/)，原版顶栏做到了这一点。侧重点是**更快**，而不是仅仅是**打开**、**找到**。

**Reply by RingingVolcano** [reply]
> 从来就不是难找的问题。
> 顺着你这逻辑，不如问问 B 站在顶栏放这么个入口（单独放外面）有什么必要？本脚本自定义顶栏**可以**将稍后再看**单独放外面**又有什么必要？也不是很难找啊。
> 
> 这不过是一个需求而已。你觉得「入口不难找」已经能满足你的需求。那楼主的需求呢？「入口不难找」就是 TA 的需求吗？
> 说实话，我自己也没这需求，我也只是顺着楼主逻辑推演而已。稍微推演共情后，我觉得需求应该是更快地打开 [番剧主页](https://www.bilibili.com/anime/)，原版顶栏做到了这一点。侧重点是**更快**，而不是仅仅是**打开**、**找到**。

顶栏作为一级入口 把番剧和主站并列不是很奇怪吗？现在移动端淡化分区 番剧入口相对还更难找了 只能说官方的顶栏只是单纯为了方便但是忽视了逻辑吧。。。
照你这么说 是不是要把每个分区都放顶栏一级入口 满足不同人的快速打开特定分区的需求？
不能完全照顾到所有人的特殊需求吧
稍后再看与收藏 历史并列 也算符合同级逻辑吧

**Reply by Geeyun-JY3** [reply]
番剧和主站在顶栏并列合不合逻辑这事，我觉得确实是不讲逻辑，不知道 B 站是不是从实用主义出发把这入口往顶栏塞的。但这构不构成不做这功能的理由，就看作者怎么想了。

> 是不是要把每个分区都放顶栏一级入口 满足不同人的快速打开特定分区的需求？

这倒确实 ~~（真要做的话要不限定可添加的分区入口数量）~~。根源还是 B 站。

**Reply by RingingVolcano** [reply]
> 
> 这倒确实 ~（真要做的话要不限定可添加的分区入口数量）~。根源还是 B 站。

所以快点出V2，模块化之后就能搞强自定义 ~~想咋折腾咋折腾~~（

**Comment by RingingVolcano** [comment]
已经收在[主站]下面了
![LBHJ$@_WZ()_9BKJ~1K4( E](https://user-images.githubusercontent.com/72140353/123606677-bb28ae00-d82f-11eb-8cac-7a232e6b259f.png)
